homeassistant/components/smarttub/climate.py

```diff
@@ -70,6 +69,8 @@ class SmartTubThermostat(SmartTubEntity, ClimateEntity):
         ClimateEntityFeature.PRESET_MODE | ClimateEntityFeature.TARGET_TEMPERATURE
     )
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
```

Comment:
`_attr_temperature_unit` is always set to `UnitOfTemperature.CELSIUS`, so there is no point in converting celsius to celsius below